release/llm_tests/serve/configs/model_config/llama_3dot1_8b_lora.yaml

```diff
@@ -7,6 +7,7 @@ accelerator_type: A10G
 engine_kwargs:
   max_model_len: 2048
   enable_lora: true
+  enforce_eager: true
```

Comment:
seems like v0 was slightly faster for model intiialization. making it eager so that model is loaded faster